### ISSUE TITLE
Make it easier to differentiate between categories, channels, playlists etc

### ISSUE THREAD

**Issue by mykhailodanilenko** [OwnTube-tv/web-client#168]
Currently on category and playlist page there is no explicit mention of the type of list you are viewing. If playlists are added to the channel page it is also unclear which is which until you look at the button saying "View playlists"/"View all", ideally it could be something like on the homepage where channels are on a colored background and categories are not.

**Comment by mkl-adsn** [comment]
Lets discuss how to best approach this one on a standup

**Comment by mkl-adsn** [comment]
@mblomdahl It took a little tweaking of the existing design to incorporate the sticky section labels but I think I've landing in a good solution. For now I focused on the start page for desktio and added the changes on a separate artboard until we have a thumbs up on the design as to not mess with the main prototype.

I also tried out the idea of having labels on each category/playlist/etc but it didn't look very good and also the button CTA text already fills to role as a label in a way, so I left that out.

Changes below, looking forwartd to feedback!

**List of changes**
- Added a title block for each section that sticks to the top of the page when scrolling
- Added buttons in the title sections to see all categories/playlists/etc. (Same core functionality as the menu but I think it's good redundancy
- Deemphasized the color on the buttons for single channels/playlists/categories to create better visual hierarcy with the new  buttons in the title sections
- Lowered font size and weight for single channel/playlist/category titles to create better visual hierarcy with the new sticky section titles
- Lowered spacing between sections a bit and added a divider to clearer separate them
- Removed channel block background color as the sticky sections now fills the need for distinguishing the content types
- Slightly repositioned global share button to not interfere with the sticky section header

Scrollable prototype: https://www.figma.com/proto/f15L42P8qyLqNkzm0Eeo8G/Owntube?page-id=&node-id=1031-12875&node-type=frame&viewport=6342%2C-9%2C0.39&t=mi2c6aP2MsAutYCG-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=1031%3A12875&show-proto-sidebar=1

[(Artboard in Figma)](https://www.figma.com/design/f15L42P8qyLqNkzm0Eeo8G/Owntube?node-id=1031-12875&t=vdB9kgNy04WroJEY-1)
